### PR TITLE
Fix typos in settings description

### PR DESCRIPTION
**Pull request by aheejin**

### CODE REVIEW

File: src/settings.js [hunk 1]
```diff
@@ -1840,8 +1840,8 @@ var DISABLE_EXCEPTION_THROWING = 0;
 var USE_OFFSET_CONVERTER = 0;
 
 // If set to 1, the JS compiler is run before wasm-ld so that the linker can
-// report undefined symbols within the binary.  Without this option that linker
-// doesn't know which symbols might be defined JS and so reporting of undefined
+// report undefined symbols within the binary.  Without this option linker
```
Comment (sbc100) [inline src/settings.js:1843]: I think this should be "the linker"

File: src/settings.js [hunk 2]
```diff
@@ -1943,12 +1943,12 @@ var SPLIT_MODULE = 0;
 // How to calculate reverse dependencies (dependencies from JS functions to
 // native functions) prior to linking native code with wasm-ld.  This option
 // has three possible values:
-// 'auto': (default) inspect the object code passed to the linker (by running
-//         llvm-nm) all all inputs and use the map in deps_info.py to
-//         determines the set of additional dependencies.
+// 'auto': (default) Inspect the object code passed to the linker (by running
+//         llvm-nm) and use the map in deps_info.py to determine the set of
```
Comment (sbc100) [inline src/settings.js:1947]: I think I meant to write: `(by running llvm-nm on all input)`

**Review by sbc100 — APPROVED** [review]

File: src/settings.js (hunk 1 above)
Comment (aheejin) [reply src/settings.js:1843]: Done

File: src/settings.js (hunk 2 above)
Comment (aheejin) [reply src/settings.js:1947]: Done

**Pull request merged**